modules/nextclade.nf

```diff
@@ -0,0 +1,13 @@
+process nextclade {
```

Comment:
It would be preferential to include this module in one of the existing module scripts. Is that possible?

Reply:
Yes, I did this on my local computer. I can update the script.

Reply:
I added this module to modules/typing.nf